### ISSUE TITLE
Vacuum image not working inside vertical-stack-in-card

### ISSUE THREAD

**Issue by jacekpaszkowski** [benct/lovelace-xiaomi-vacuum-card#28]
Since some days i have a problem with casd placed inside vertical-stack-in-card.
Vacuum image is not loading. But works when to placed inside vertical-stack-in-card image also loads in design view when placed inside vertical-stack-in-card

```
cards:
  - buttons:
      locate: true
      pause: true
      return: true
      spot: false
      start: false
      stop: true
    entity: vacuum.xiaomi_vacuum_cleaner
    image: /local/images/xiaomi_vacuum_2.png
    labels:
      battery: Bateria
      filter: Filtr
      main_brush: Główna szczotka
      mode: Moc
      sensor: Sensory
      side_brush: Boczna szczotka
      status: Status
    name: Rosie
    type: 'custom:xiaomi-vacuum-card'
  - entities:
      - entity: sensor.vacuum_time_from_last_cleanup
        name: Czas od
      - entity: sensor.vacuum_cleaning_time
        name: Trwało
      - entity: sensor.vacuum_cleaned_area
        name: Powierzchnia
    type: glance
  - entities:
      - entity: input_select.vacuum_power
      - entity: input_select.vacuum_room
      - entity: script.trigger_automatic_cleanup
    type: entities
  - cards:
      - color: var(--paper-item-icon-active-color)
        color_off: var(--paper-item-icon-color)
        color_type: icon
        dbltap_action: none
        entity: automation.cleaning_timer_weekdays
        hold_action: none
        show_name: true
        show_state: false
        size: 40px
        tap_action:
          action: call-service
          service: homeassistant.toggle
          service_data:
            entity_id: automation.cleaning_timer_weekdays
        type: 'custom:button-card'
      - color: var(--paper-item-icon-active-color)
        color_off: var(--paper-item-icon-color)
        color_type: icon
        dbltap_action: none
        entity: automation.cleaning_timer_weekend
        hold_action: none
        show_name: true
        show_state: false
        size: 40px
        tap_action:
          action: call-service
          service: homeassistant.toggle
          service_data:
            entity_id: automation.cleaning_timer_weekend
        type: 'custom:button-card'
    type: horizontal-stack
  - cards:
      - card:
          entities:
            - sensor.vacuum_start_time_weekdays
            - input_number.vacuum_start_hour_weekdays
            - input_number.vacuum_start_minutes_weekdays
          type: entities
        conditions:
          - entity: automation.cleaning_timer_weekdays
            state: 'on'
        type: conditional
      - card:
          entities:
            - sensor.vacuum_start_time_weekend
            - input_number.vacuum_start_hour_weekend
            - input_number.vacuum_start_minutes_weekend
          type: entities
        conditions:
          - entity: automation.cleaning_timer_weekend
            state: 'on'
        type: conditional
    type: vertical-stack
type: 'custom:vertical-stack-in-card'
```

![image](https://user-images.githubusercontent.com/4264991/72369101-afaafb00-36ff-11ea-8175-aad0ce26c92a.png)

**Comment by talondnb** [comment]
Same issue here.

**Comment by vMeph** [comment]
same issue

**Comment by collse** [comment]
sam here - waiting for fix

**Comment by talondnb** [comment]
I separated the card from the vertical stack with other entities and it works fine. Doesn't seem to like stacking with other entities.

**Comment by collse** [comment]
> I separated the card from the vertical stack with other entities and it works fine. Doesn't seem to like stacking with other entities.
> 
> EDIT: It used to work fine with other entities though..

exactly what I did until a fix is present, moved it out of the vertical card which works fine, embedded in the vertical card it doesnt query for the image file which worked fine previously as you mention

**Comment by bremor** [comment]
I was experiencing this bug also and came up with a temporary fix... I am not a web programmer by any means so this is probably not the right way to fix this. It works but will be overwritten if you update the card within HACS. Update the js file within the .gz file (Open using 7zip or similar)
**Old**
```
        .background {
          background-repeat: no-repeat;
          background-position: center center;
          background-size: cover;
        }
```
**New**
```
        .background {
          background-repeat: no-repeat !important;
          background-position: center center !important;
          background-size: cover !important;
          background-image: url(/local/img/vacuum.png) !important;
        }
```

**Comment by benct** [comment]
This is not actually a bug in this card, but a problem with `vertical-stack-in-card` itself, as reported in the above-mentioned issue. Please follow up on the issue there instead.

**Comment by jacekpaszkowski** [comment]
@benct could you please create this issue in vertical-stack-in-card repo? It would be great because you've already made some research. Sou you can point the problem :)

**Comment by benct** [comment]
Already exists ([#71](https://github.com/custom-cards/vertical-stack-in-card/issues/71)), no point making a duplicate issue.

**Comment by jacekpaszkowski** [comment]
@bremor of course, i've missed your commnet there :) great, thank you!

**Comment by mateuszjestfajny** [comment]
> I was experiencing this bug also and came up with a temporary fix... I am not a web programmer by any means so this is probably not the right way to fix this. It works but will be overwritten if you update the card within HACS. Update the js file within the .gz file (Open using 7zip or similar)
> **Old**
> 
> ```
>         .background {
>           background-repeat: no-repeat;
>           background-position: center center;
>           background-size: cover;
>         }
> ```
> 
> **New**
> 
> ```
>         .background {
>           background-repeat: no-repeat !important;
>           background-position: center center !important;
>           background-size: cover !important;
>           background-image: url(/local/img/vacuum.png) !important;
>         }
> ```

Where did You change that?

**Comment by benct** [comment]
This should now be fixed in version `0.3.0` of `vertical-stack-in-card`.
